extensions/objects/templates/filepath-editor.component.ts

```diff
@@ -25,8 +25,12 @@ import { downgradeComponent } from '@angular/upgrade/static';
   styleUrls: []
 })
 export class FilepathEditorComponent implements OnInit {
-  @Input() modalId;
-  @Input() value;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() modalId!: symbol;
+  // 'value' is undefined when filepath is empty.
+  @Input() value: string | undefined;
```

Comment:
Are we assigning `undefined` to `value` anywhere ?

Reply:
no, but the value can be empty in the form so I think this case should be catered

Reply:
I think that ` | undefined` would mean that we're assigning an undefined value to the property. Doing non-null assertion should be fine.

Reply:
done